crates/oxc_ast/src/serialize.rs

```diff
@@ -538,6 +571,9 @@ impl ESTree for ClassImplements<'_, '_> {
     raw_deser = "
         const ident = DESER[Box<IdentifierReference>](POS);
         ident.type = 'JSXIdentifier';
+        delete ident.decorators;
+        delete ident.optional;
+        delete ident.typeAnnotation;
```

Comment:
I went with this to counter against `IdentifierReference` raw deser for `JSXElementIdentifierReference`.

Reply:
I'll try and make that a bit faster in a follow-up.